lib/lotus/utils/kernel.rb

```diff
@@ -418,8 +426,9 @@ def self.Integer(arg)
       #   # Missing #respond_to?
       #   input = BasicObject.new
       #   Lotus::Utils::Kernel.BigDecimal(input) # => TypeError
-      def self.BigDecimal(arg)
+      def self.BigDecimal(arg, precision: DEFAULT_PRECISION)
         case arg
+        when ->(a) { a.respond_to?(:to_d) && a.class == Rational } then arg.to_d(precision)
```

Comment:
@AlfonsoUceda Hey thanks for looking at this.

1. `DEFAULT_PRECISION` it isn't a descriptive name. We know that is about BigDecimal, but only because we're discussing in this PR. By looking at that, there is no mention of this. What if we need to introduce a precision for Float? Shall we specify `BIGDECIMAL_DEFAULT_PRECISION` instead?
2. Applying `precision:` only to Rational is misleading. From an outside perspective would expect that precision is always applied, regardless of the input. Can we apply for all the kind of inputs? Which precision should we choose by default?